mock/world/builtinFunctionsMock.go

```diff
@@ -133,5 +132,4 @@ func addOutputTransferToVMOutput(
 		Address:         recipient,
 		OutputTransfers: []vmcommon.OutputTransfer{outTransfer},
 	}
-	vmOutput.GasRemaining = 0
```

Comment:
:+1: 

Unrelated to this line, but pertaining to the entire function `addOutputTransferToVMOutput()`: consider changing this function to accept a `vmInput` directly and an additional boolean `isSCCallAfter` to instruct the function to recompute the `endpoint`.